src/cfn/template.yaml

```diff
@@ -453,6 +435,57 @@ Resources:
 
   # Computing & Containers
 
+  AWSSDKLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      LayerName: MediaAnalysisAWSSDK
+      Description: Latest confirmed compatible AWS SDK
+      ContentUri: ../backend/functions/layers/aws_sdk/
+      CompatibleRuntimes:
+        - nodejs8.10
+        - nodejs10.x
+
+  FPLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      LayerName: MediaAnalysisFP
+      Description: Layer containing functional programming libs
+      ContentUri: ../backend/functions/layers/fp/
+      CompatibleRuntimes:
+        - nodejs8.10
+        - nodejs10.x
+
+  Orchestrator:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs10.x
+      CodeUri: ../backend/functions/orchestrator/dist/
+      Timeout: 10
```

Comment:
Is 10 enough? I am thinking of the scenario of Fargate stopping the container by sending the SIG_TERM and allowing the app to gracefully save to dynamo, finish up and all the things.

Reply:
yeah I just took whatever the existing deployed timeout was but thinking about it you're right - we should at least have the default stop task timeout as a minimum which is 30 secs. so maybe 1 min to be safe?

Reply:
yeah, 60 sounds like a good amount